personalities/sctfe/ct_server_gcp/main.go

```diff
@@ -290,6 +290,7 @@ func newGCPStorage(ctx context.Context, vCfg *sctfe.ValidatedLogConfig, signer n
 		return nil, fmt.Errorf("Failed to initialize GCP issuer storage: %v", err)
 	}
 
+	// TODO: repalce with a global dedup storage for GCP
```

Comment:
typo: repalce

Reply:
Done already,I think you might have reviewed an older version of this file.

Reply:
Ah yes, I went commit-by-commit, sorry for the noise!